src/credentials/PersistentStorageOpCertStore.cpp

```diff
@@ -255,24 +254,19 @@ CHIP_ERROR PersistentStorageOpCertStore::AddNewOpCertsForFabric(FabricIndex fabr
     ReturnErrorCodeIf(StorageHasCertificate(mStorage, fabricIndex, CertChainElement::kNoc), CHIP_ERROR_INCORRECT_STATE);
     ReturnErrorCodeIf(StorageHasCertificate(mStorage, fabricIndex, CertChainElement::kIcac), CHIP_ERROR_INCORRECT_STATE);
 
-    Platform::ScopedMemoryBuffer<uint8_t> nocBuf;
+    Platform::ScopedMemoryBufferWithSize<uint8_t> nocBuf;
```

Comment:
@bzbarsky-apple Is this what you meant re: using `ScopedMemoryBufferWithSize` ?

See https://github.com/project-chip/connectedhomeip/pull/19643#discussion_r899761501

*PLEASE* do not merge until this comment is resolved by @bzbarsky-apple

Reply:
Yes, this is what I meant.